src/secret-dj/operations/createGame.ts

```diff
@@ -27,35 +27,34 @@ export const createGame = async ({
 		const box = await db.box.create({
 			data: {
 				name: `secret dj/${name}`,
-				ownerId: emailId,
 				stylesheet: `https://${Config.HOST}/public/secret-dj/styles.css`,
 			},
 			select: {
 				id: true,
 			},
 		});
 
-		await db.subscription.create({
-			data: { boxId: box.id, emailId },
-		});
+		const [, , result] = await Promise.all([
+			db.subscription.create({
+				data: { boxId: box.id, emailId },
+			}),
+			db.permission.create({
+				data: { boxId: box.id, emailId, canDelete: true, canKill: true },
+			}),
+			db.season.create({
+				data: {
+					name,
+					ruleCount,
+					ownerId,
+					description,
+					boxId: box.id,
+					unlisted,
+				},
+				select: {
+					id: true,
+				},
+			}),
+		]);
```

Comment:
lmao is this pattern what prompted the max jobs stuff

Reply:
lol no this here is fine. it's just i am afraid of a LOT of replies coming in all at once